### ISSUE TITLE
graph-sycn failing for package-extract task

### ISSUE THREAD

**Issue by harshad16** [thoth-station/graph-sync-job#562]
**Describe the bug**
Failing for graph-sync of package-extract task.

```
Traceback (most recent call last):
  File "app.py", line 252, in <module>
    cli()
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 1066, in invoke
{"name": "sentry_sdk.errors", "levelname": "DEBUG", "module": "transport", "lineno": 222, "funcname": "_send_event", "created": 1615578073.506125, "asctime": "2021-03-12 19:41:13,506", "msecs": 506.12497329711914, "relative_created": 9300.085306167603, "process": 1, "message": "Sending event, type:null level:error event_id:f520c00c51a241b9a5f19fdb62db14d6 project:1298083 host:sentry.io"}
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "app.py", line 226, in cli
    _do_sync(
  File "app.py", line 125, in _do_sync
    stats = sync_documents(
  File "/opt/app-root/lib64/python3.8/site-packages/thoth/storages/sync.py", line 574, in sync_documents
    stats_change = handler(
  File "/opt/app-root/lib64/python3.8/site-packages/thoth/storages/sync.py", line 232, in sync_analysis_documents
    graph.sync_analysis_result(document)
  File "/opt/app-root/lib64/python3.8/site-packages/thoth/storages/graph/postgres.py", line 4674, in sync_analysis_result
    raise ValueError("Python Version does not match pattern")
ValueError: Python Version does not match pattern
```

**Comment by harshad16** [comment]
This has occurred as i have submitted an non-compatible image for analysis via managment-api.
https://github.com/thoth-station/storages/blob/ce0734d4f00789aa48e0b656f6649eb2369192cc/thoth/storages/graph/postgres.py#L4742
`is_external` the key here
https://github.com/thoth-station/management-api/blob/1eaa32e846e50f4a598b41fe0c40f956e605e32d/thoth/management_api/api_v1.py#L295

**Comment by fridex** [comment]
I think this can be closed. The issue should be fixed by  thoth-station/storages#2244 and the most recent release should have the fix addressed https://github.com/thoth-station/graph-sync-job/pull/567. Thanks @harshad16 for this :+1:
